contracts/test/PoolClaim.t.sol

```diff
@@ -515,6 +554,68 @@ contract PoolTest is TestUtil {
         return newAmt == uint256(0) ? uint256(0) : newAmt.mul(totalClaim).div(totalNewAmt);
     }
 
+    function getFundedAmount(uint256 val) internal pure returns(uint256) {
+        return constrictToRange(val, 100 * USD,  val);
+    }
+
+    function getDL(Pool pool, Loan loan, DebtLockerFactory dlFactory) internal view returns(address) {
+        return pool.debtLockers(address(loan),   address(dlFactory));
+    }
+
+    function generateLoanAndFundedAmount(
+        uint256 apr,
+        uint256 index,
+        uint256 numPayments,
+        uint256 requestAmount,
+        uint256 collateralRatio
+    ) internal returns(uint256 fundedAmount) {
+        uint256[5] memory specs = getFuzzedSpecs(apr, index, numPayments, requestAmount, collateralRatio);
+        address[3] memory calcs = [address(repaymentCalc), address(lateFeeCalc), address(premiumCalc)];
+
+        loan  = bob.createLoan(address(loanFactory), USDC, WETH, address(flFactory), address(clFactory), specs, calcs);
+        loan2 = ben.createLoan(address(loanFactory), USDC, WETH, address(flFactory), address(clFactory), specs, calcs);
+
+        fundedAmount = constrictToRange(specs[3], 100 * USD, 1E10 * USD, true);  // Fund value should be between 100 USD - 1B USD
```

Comment:
We can get rid of the if statement below if we just use `requestAmount = specs[3]`

That way we dont need any of the logic below. We would be deposing 10x the requestAmount for these two loans.

Reply:
But below code will only kick in when the value is less than the `specs[3]` otherwise we can deal the large value.